anime_downloader/config.py

```diff
@@ -128,6 +128,10 @@
                 "gounlimited"
             ]
         },
+        'dbanimes': {
+            'server': 'fembed',
+            'fallback_servers': ['mixdrop', 'sendvid']
```

Comment:
I suggest using sort_sources instead, or at least in the future for less headaches with selected servers and all that.